apollo-runtime/src/main/java/com/apollographql/apollo/cache/normalized/StandardCacheHeaders.java

```diff
@@ -0,0 +1,14 @@
+package com.apollographql.apollo.cache.normalized;
+
+/**
+ * A collection of "standardized" cache headers that an implementation {@link NormalizedCache}
+ * is recommended to respect.
+ */
+public class StandardCacheHeaders {
+
+  /**
+   * Records from this request should not be stored in the {@link NormalizedCache}.
+   * This does **not** specify that a request should not be read from a cache.
+   */
+  public static final String DO_NOT_CACHE = "do-not-cache";
```

Comment:
Seems the right thing to change the name to `CacheFetchStrategy` plus I would like to merge `HttpCacheControl` with `CacheControl` if it possible, what do you think?